Source/Eto.WinForms/WinConversions.cs

```diff
@@ -403,7 +403,7 @@ public static MouseButtons ToEto(this swf.MouseButtons button)
 
 		public static Graphics ToEto(this sd.Graphics g)
 		{
-			return new Graphics(new GraphicsHandler(g));
+			return new Graphics(new GraphicsHandler(g, false));
```

Comment:
Hi @firda-cze, thanks for taking the time to think about how this could be done better.  I agree that both `ToEto` and `GraphicsHandler` ctor should have a non-default bool parameter to specify ownership there, which will make things more explicit.